pcf-tile-metadata/aci-cni-plugin.yml.template

```diff
@@ -0,0 +1,377 @@
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_app_static_ext_ip_pool_start"
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_app_static_ext_ip_pool_end"
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_ip_pool_start"
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_ip_pool_end"
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_subnets"
+    label: Network node service subnets
+    description: data_type:string_list 
+  - reference: ".properties.networking_aci_api_port" 
+    label: Networking ACI Api Port
+    description: date_type:port
+  - reference: ".properties.network_subnet" 
+    label: Network Subnet
+    description: date_type:string 
+
+runtime_configs:
+- name: aci-containers-tile 
+  runtime_config: |
+    releases: 
+    - name: aci-containers
+      version: '<aci-containers-release-version>'
+    addons:
+    - name: network-setup-addon 
+      jobs:
+      - name: network-setup
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+    - name: external-cni-path-setup-addon  
+      jobs:
+      - name: external-cni-path-setup 
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+    - name: consul-dns-setup-addon
+      jobs: 
+      - name: consul-dns-setup  
+        release: aci-containers
+      include:
+        jobs:
+        - name: gorouter
+          release: routing
+        stemcell:
+        - os: ubuntu-trusty
+    - name: openvswitch-switch-addon 
+      jobs:
+      - name: openvswitch-switch
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+    - name: agent-ovs-addon 
+      jobs:
+      - name: agent-ovs
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+      properties:
+        apic:
+          vmm_policy: (( .properties.apic_vmm_policy.value ))
+          vmm_domain: (( .properties.apic_vmm_domain.value ))
+          vmm_controller: (( .properties.apic_vmm_controller.value ))
+          encap_type: (( .properties.apic_encap_type.value ))
+    - name: aci-cf-host-agent-addon
+      jobs:
+      - name: aci-cf-host-agent
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+      properties:
```

Comment:
Please recheck the job properties as specified in latest version of cf_ops.yml, I believe there are changes that are not incorporated here.